data-serving/data-service/.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  semi: true,
+  trailingComma: "all",
+  singleQuote: true,
+  printWidth: 120,
```

Comment:
but, but, our dear and treasured 80 col limit??!!?

https://google.github.io/styleguide/jsguide.html#formatting-column-limit

just kidding, 120 lgtm!

Reply:
Ah, good catch! I thought I ought to change this, but forgot to circle back.

Prettier recommends 80, too -- so we're officially correct.

https://prettier.io/docs/en/options.html#print-width